bootstrap/cmd/bootstrap/app/options/options.go

```diff
@@ -47,6 +48,7 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 	fs.BoolVar(&s.JsonLogFormat, "json-log-format", true, "Set true to use json style log format. Set false to use plaintext style log format")
 	fs.IntVar(&s.Port, "port", 8080, "The port to use when running an http server.")
 	fs.StringVar(&s.AppDir, "app-dir", "/opt/bootstrap", "The directory for the ksonnet applications.")
+	fs.StringVar(&s.GkeVersion, "GkeVersion", "1.10", "Initial GKE master version")
```

Comment:
The default should be the empty string so that we use the value in the DM config by default.